packages/toolkit/src/dao/apps.js

```diff
@@ -1,40 +1,37 @@
+import { takeWhile } from 'rxjs/operators'
 import { abi as kernelAbi } from '@aragon/abis/os/artifacts/Kernel'
 //
-import {
-  initAragonJS,
-  getApps,
-  resolveEnsDomain,
-} from '../helpers/aragonjs-wrapper'
+import { resolveEnsDomain } from '../helpers/resolveAddressOrEnsDomain'
+import { useEnvironment } from '../helpers/useEnvironment'
+import { initWrapper } from '../helpers/wrapper'
 
 /**
  * Return installed apps for a DAO
  *
  * @param {string} dao DAO address
- * @param {Object} options Options
```

Comment:
Remove the object inputs across all functions and instead the API ask the needed information for each particular funtionality and the envrionment option.